File: Source/Web/SS.StudentStore.UI/src/app/model/product/producttypemapping.ts
export interface IProducttypeMapping{
    productTypeMappingId: number;
    productTypeId: number;
    productId: number;
    name: string;
    desctiption: string;
    price: number;
}

export class ProductTypeMapping implements IProducttypeMapping{
    productTypeMappingId: number;
    productTypeId: number;
    productId: number;
    name: string;
    desctiption: string;
    price: number;
    
}

File: Source/Services/SS.StudentStore.Services/SS.StudentStore.Services.Core/Entities/ProductTypeMapping.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace SS.StudentStore.Services.Core.Entities
{
    public class ProductTypeMapping
    {
        [Key]
        public long ProductTypeMappingId { get; set; }
        public int ProductTypeId { get; set; }
        public long ProductId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        public bool IsActive { get; set; }
    }
}
